app/components/shared/review_component/review_component.html.slim

```diff
@@ -0,0 +1,13 @@
+.review-component
+  .review-component__heading
+    h2.govuk-heading-m.review-component__section id="#{header.id}_heading"
+      = header.title
+      = govuk_link_to(t("buttons.change"), header.edit_link, class: "review-component__section-button", "aria-label": t("jobs.aria_labels.change_job_location"))
+
+  - if body.present?
+    .review-component__body
+      = render GovukComponent::SummaryList.new do |component|
```

Comment:
I think the component might be better suited to just the header, and then we use the relevant govuk-component in the view (and pass in any classes or html_attributes). The designs I've seen show review pages with a mixture of summary lists and accordions - so I don't think we gain anything by passing in quite a messy array into the component just to use it to render a summary list or accordion.

Reply:
@tv_devs above has negligable pluses and minus, let me know your thoughts, first thumbs up or down on connors comment decides - apart from connor ;)